deploy/roles/apache/tasks/main.yml

```diff
@@ -50,19 +49,19 @@
 
 - name: Open port 80 in iptables
   command: "iptables -I {{ iptables_chain }} 3 -m state --state NEW -p tcp --dport 80 -j ACCEPT"
-  when: iptables_config
+  when: iptables_config | bool # [DEPRECATION WARNING]: evaluating iptables_config as a bare variable
   tags:
     - apache
 
 - name: Open port 443 in iptables
   command: "iptables -I {{ iptables_chain }} 3 -m state --state NEW -p tcp --dport 443 -j ACCEPT"
-  when: iptables_config
+  when: iptables_config == True # [DEPRECATION WARNING]: evaluating iptables_config as a bare variable
   tags:
     - apache
 
 - name: Save rules
   command: "/sbin/service iptables save"
-  when: iptables_config
+  when: iptables_config | default(false) | bool # [DEPRECATION WARNING]: evaluating iptables_config as a bare variable
```

Comment:
This warning appears 40 times across deployment because I am using ansible version 2.8, not 2.5. This was my attempt to fix it. Should I apply this fix to all warnings?